frontend/src/component/common/Badge/Badge.tsx

```diff
@@ -121,12 +110,15 @@ export const Badge: FC<IBadgeProps> = forwardRef(
         >
             <ConditionallyRender
                 condition={Boolean(icon) && !iconRight}
-                show={BadgeIcon(color, icon!, false, Boolean(children))}
+                show={BadgeIcon(color, icon!)}
+            />
+            <ConditionallyRender
+                condition={Boolean(children)}
+                show={<div>{children}</div>}
```

Comment:
Why are we putting the children within a div here? Couldn't this whole component be replaced with `{children}`? Is it in case we have several children and we need to group them?

Reply:
Because `children` can be <Fragment/> containing multiple elements, and without wrapping it we can run into issues where flexbox adds spacing between every `child` item